types/cross-spawn/cross-spawn-tests.ts

```diff
@@ -8,13 +8,13 @@ pw.on('error', (err: Error) => {
     console.log(err);
 });
 
-const r1: Error = spawn.sync('foo').error;
-const r2: string[] = spawn.sync('foo').output;
-const r3: Buffer = spawn.sync('foo').stdout;
-const r4: Buffer = spawn.sync('foo').stderr;
-const r5: number = spawn.sync('foo').status;
-const r6: string = spawn.sync('foo').signal;
-const r7: string = spawn.sync('foo', {encoding: 'utf8'}).stdout;
+const r1: Error | null = spawn.sync('foo').error;
+const r2: [Buffer | null, Buffer | null, Buffer | null] | null = spawn.sync('foo').output;
+const r3: Buffer | null = spawn.sync('foo').stdout;
+const r4: Buffer | null = spawn.sync('foo').stderr;
+const r5: number | null = spawn.sync('foo').status;
+const r6: string | null = spawn.sync('foo').signal;
+const r7: string | null = spawn.sync('foo', {encoding: 'utf8'}).stdout;
 
```

Comment:
can you also add a test to ensure the type is correctly inferred after testing if `error == null`?

Reply:
Done.